hypervisor/acpi_parser/acpi_ext.c

```diff
@@ -143,4 +156,8 @@ void acpi_fixup(void)
 			rr_data->val = *(facp_addr + OFFSET_RESET_VALUE);
 		}
 	}
+
+	mcfg_addr = (uint8_t *)get_acpi_tbl(ACPI_SIG_MCFG);
+	def_mmcfg_base = parse_mmcfg_base(mcfg_addr);
+	set_mmcfg_base(def_mmcfg_base);
```

Comment:
acpi_fixup never fails (return void), while this API means it may fail (panic).  Should we use consistant way?

Reply:
hi Eddie,
I've updated the patch, please review.